src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/WorkspaceSemanticTokensRefreshNotifier.cs

```diff
@@ -37,10 +36,24 @@ public WorkspaceSemanticTokensRefreshNotifier(
 
         _disposeTokenSource = new();
 
+        _queue = new(
+            TimeSpan.FromMilliseconds(250),
+            ProcessBatchAsync,
+            _disposeTokenSource.Token);
+
         _isColoringBackground = optionsMonitor.CurrentValue.ColorBackground;
         _optionsChangeListener = optionsMonitor.OnChange(HandleOptionsChange);
     }
 
+    private ValueTask ProcessBatchAsync(CancellationToken token)
+    {
+        _clientConnection
+            .SendNotificationAsync(Methods.WorkspaceSemanticTokensRefreshName, _disposeTokenSource.Token)
```

Comment:
`token` should be passed here instead of `_disposeTokenSource.Token`.

Reply:
oh woops, good catch